srcts/src/bindings/input/selectInput.ts

```diff
@@ -119,18 +120,30 @@ class SelectInputBinding extends InputBinding {
       $el
         .parent()
         .find('script[data-for="' + $escape(el.id) + '"]')
-        .replaceWith(data.config);
+        .replaceWith(data.config as NonNullable<typeof data.config>);
       this._selectize(el, true);
     }
 
     // use server-side processing for selectize
     if (hasOwnProperty(data, "url")) {
-      selectize = this._selectize(el);
+      selectize = this._selectize(el) as SelectizeInfo;
       selectize.clearOptions();
       let loaded = false;
 
-      selectize.settings.load = function (query, callback) {
-        const settings = selectize.settings;
+      type CallbackFn = Parameters<
+        NonNullable<SelectizeInfo["settings"]["load"]>
+      >[1];
+      const innerSelectize = selectize as SelectizeInfo & {
```

Comment:
The remainder of this method uses `innerSelectize` as TypeScript did not like that `selectize.settings.load` will always exist. Line 136 should only be a re-typing and should be a no-op when built.

Reply:
Maybe move the type definition up to 129, then tsc will know settings.load exists for this branch?